caleb-lab/controllers/note-controller.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
```

Comment:
**TODO FOR LOST POINTS**

Missing these:

include an .eslintrc
include a .gitignore
include a readme with a project description